manifests/07_deployment.yaml

```diff
@@ -39,6 +39,10 @@ spec:
         image: quay.io/openshift/origin-openstack-cinder-csi-driver-operator:latest
         imagePullPolicy: IfNotPresent
         name: openstack-cinder-csi-driver-operator
+        resources:
+          requests:
+            cpu: 10m
+            memory: 20Mi
```

Comment:
I think we need to fix the manifests in `./assets` instead. Those are the resources that the operator deploys in the cluster.

Reply:
Suggestion: check the EBS operator here: https://github.com/openshift/aws-ebs-csi-driver-operator

Reply:
Thanks for the review @bertinatto.

I've fixed the manifests in https://github.com/openshift/cluster-storage-operator/pull/108 and fixed them here too while I was at it.

I'm not sure I understand your comment about `./assets` - could you clarify what needs to be fixed in the assets directory?

Reply:
Sorry, I meant to say we should also update files in this directory:

https://github.com/openshift/openstack-cinder-csi-driver-operator/tree/master/assets

This will fix the QoS for the controller and the node shown in this message:

https://github.com/openshift/cluster-storage-operator/pull/108#issue-531912096

We can keep the changes in this file (`manifests/07_deployment.yaml`), but you should also update the file in CSO (as you're correctly doing in PR https://github.com/openshift/cluster-storage-operator/pull/108/files). This particular file is here just for reference as it's not used by anything.

Reply:
Gotcha! I'll update accordingly.